README.md

```diff
@@ -28,68 +28,23 @@ Features not supported:
 
 ## Wallet API
 
-Constructors:
-
-- `generate([icap])` - create an instance based on a new random key (setting `icap` to true will generate an address suitable for the `ICAP Direct mode`)
-- `generateVanityAddress(pattern)` - create an instance where the address is valid against the supplied pattern (**this will be very slow**)
-- `fromPrivateKey(input)` - create an instance based on a raw private key
-- `fromExtendedPrivateKey(input)` - create an instance based on a BIP32 extended private key (xprv)
-- `fromPublicKey(input, [nonStrict])` - create an instance based on a public key (certain methods will not be available)
-- `fromExtendedPublicKey(input)` - create an instance based on a BIP32 extended public key (xpub)
-- `fromV1(input, password)` - import a wallet (Version 1 of the Ethereum wallet format)
-- `fromV3(input, password, [nonStrict])` - import a wallet (Version 3 of the Ethereum wallet format). Set `nonStrict` true to accept files with mixed-caps.
-- `fromEthSale(input, password)` - import an Ethereum Pre Sale wallet
-
-For the V1, V3 and EthSale formats the input is a JSON serialized string. All these formats require a password.
-
-Note: `fromPublicKey()` only accepts uncompressed Ethereum-style public keys, unless the `nonStrict` flag is set to true.
-
-Instance methods:
-
-- `getPrivateKey()` - return the private key
-- `getPublicKey()` - return the public key
-- `getAddress()` - return the address
-- `getChecksumAddressString()` - return the [address with checksum](https://github.com/ethereum/EIPs/issues/55)
-- `getV3Filename([timestamp])` - return the suggested filename for V3 keystores
-- `toV3(password, [options])` - return the wallet as a JSON string (Version 3 of the Ethereum wallet format)
-
-All of the above instance methods return a Buffer or JSON. Use the `String` suffixed versions for a string output, such as `getPrivateKeyString()`.
-
-Note: `getPublicKey()` only returns uncompressed Ethereum-style public keys.
+For information about the Wallet's API, please go to [./docs/classes/wallet.md](./docs/classes/wallet.md).
 
 ## Thirdparty API
 
 Importing various third party wallets is possible through the `thirdparty` submodule:
 
-`var thirdparty = require('ethereumjs-wallet/thirdparty')`
-
-Constructors:
+`const thirdparty = require('ethereumjs-wallet/thirdparty')`
 
-- `fromEtherCamp(passphrase)` - import a brain wallet used by Ether.Camp
-- `fromEtherWallet(input, password)` - import a wallet generated by EtherWallet
-- `fromKryptoKit(seed)` - import a wallet from a KryptoKit seed
-- `fromQuorumWallet(passphrase, userid)` - import a brain wallet used by Quorum Wallet
+Please go to [./docs/README.md](./docs/README.md) for more info.
```

Comment:
I think we also need to make it more transparent in the new docs that the integrated methods are associated with the `Thirdparty` submodule, these are now all listed under `Functions`. And for my taste the implicit references given by the "Defined in `thirdparty.ts:XXX`" texts are not enough.

Probably difficult to change the overall structure there. I would suggest to add a qualifier to the description texts, like "Thirdparty API: Import a wallet from a KryptoKit seed".

Reply:
I agree with you. I don't like how typedoc structures things. 

I'll add those comments.